src/core/request-body-search.js

```diff
@@ -315,6 +315,20 @@ class RequestBodySearch {
         return this;
     }
 
+    /**
+     * The `track_total_hits` parameter allows you to control how the total number of hits
+     * should be tracked.
+     *
+     * Pass `true`, `false`, or the upper limit (default: `10000`) of hits you want tracked.
+     *
```

Comment:
Could you add a note mentioning that this parameter was added in elasticsearch@7?

Reply:
@sudo-suhas done.